### ISSUE TITLE
Unable to reproduce the results of PackNet-SAN on DDAD

### ISSUE THREAD

**Issue by zhangjiannan1** [TRI-ML/packnet-sfm#167]
Here are the results of my training on DDAD：
|*********************************************************************************************|
| E: 32 BS: 2 - SemiSupCompletionModel                              LR (Adam): Depth 5.00e-05 |
|*********************************************************************************************|
|     METRIC     | abs_rel  | sqr_rel  |   rmse   | rmse_log |    a1    |    a2    |    a3    |
|*********************************************************************************************|
| *** /DDAD/ddad_train_val/ddad.json/val (camera_01:)                                         |
|*********************************************************************************************|
| DEPTH          |  0.155   |  3.528   |  14.022  |  0.240   |  0.804   |  0.924   |  0.967   |
| DEPTH_PP       |  0.154   |  3.425   |  13.910  |  0.238   |  0.805   |  0.925   |  0.967   |
| DEPTH_GT       |  0.148   |  3.050   |  13.874  |  0.235   |  0.812   |  0.926   |  0.969   |
| DEPTH_PP_GT    |  0.147   |  2.972   |  13.770  |  0.233   |  0.814   |  0.927   |  0.969   |
|*********************************************************************************************|

Can you provide some advice or help，thanks！

**Comment by VitorGuizilini-TRI** [comment]
Are these depth predicted or completed results? Can you please provide both?

**Comment by zhangjiannan1** [comment]
This is the result of deep completion：

| *** /DDAD/ddad_train_val/ddad.json/val (camera_01:lidar)                                    |
|*********************************************************************************************|
|*********************************************************************************************|
| DEPTH          |  0.223   |  4.379   |  15.935  |  0.294   |  0.728   |  0.883   |  0.947   |
| DEPTH_PP       |  0.217   |  4.071   |  15.747  |  0.288   |  0.734   |  0.884   |  0.948   |
| DEPTH_GT       |  0.195   |  3.952   |  17.328  |  0.297   |  0.714   |  0.880   |  0.944   |
| DEPTH_PP_GT    |  0.191   |  3.759   |  17.192  |  0.292   |  0.718   |  0.883   |  0.946   |
|*********************************************************************************************|

The strange thing is that the result of completion is worse than predicted.

Note: I only trained "camera_01"，and I add the  “input_depth_type: ['lidar']” in “datasets:train:” . I’m not sure if this must be added, but I noticed this attribute in kitti’s training profile.

here is my dataset config:
datasets:
    augmentation:
        image_shape: (384, 640)
    train:
        batch_size: 2
        num_workers: 16
        dataset: ['DGP']
        path: ['/DDAD/ddad_train_val/ddad.json']
        split: ['train']
        depth_type: ['lidar']
        input_depth_type: ['lidar']
        cameras: [['camera_01']]
        repeat: [1]
    validation:
        num_workers: 8
        dataset: ['DGP']
        path: ['/DDAD/ddad_train_val/ddad.json']
        split: ['val']
        depth_type: ['lidar']
        input_depth_type: ['']
        cameras: [['camera_01']]
    test:
        num_workers: 8
        dataset: ['DGP']
        path: ['/DDAD/ddad_train_val/ddad.json']
        split: ['val']
        depth_type: ['lidar']
        input_depth_type: ['','lidar']
        cameras: [['camera_01'],['camera_01']]

**Comment by RobinhoodKi** [comment]
Do you guys tried to train PackNet-sfm from scratch ?  I tried three times but still can not reproduce the results especially abs_rel to 0.173 . I can only obtain value between 0.2 and 0.21 . Can you guys reproduce it ? @ @zhangjiannan1

**Comment by zhangjiannan1** [comment]
> Do you guys tried to train PackNet-sfm from scratch ? I tried three times but still can not reproduce the results especially abs_rel to 0.173 . I can only obtain value between 0.2 and 0.21 . Can you guys reproduce it ? @ @zhangjiannan1

I haven't tried it, but have you tried adjusting the learning rate？

**Comment by zhangjiannan1** [comment]
![15616462084436480_rgb](https://user-images.githubusercontent.com/87403577/126094102-6bea0f6d-43b3-49db-ab95-94fa1172bb02.png)
![15616462084436480_viz](https://user-images.githubusercontent.com/87403577/126094105-ba9aade2-5dad-4365-be92-1e191febb6d1.png)
This is my result, look forward to your help.

**Comment by RobinhoodKi** [comment]
Do you have a wechat ? we can have a discussion and share more details. My wechat id is j1327575842.

**Comment by baiyancheng20** [comment]
> Here are the results of my training on DDAD： |_********************************************************************************************| | E: 32 BS: 2 - SemiSupCompletionModel LR (Adam): Depth 5.00e-05 | |********************************************************************************************_| | METRIC | abs_rel | sqr_rel | rmse | rmse_log | a1 | a2 | a3 | |_********************************************************************************************| | *** /DDAD/ddad_train_val/ddad.json/val (camera_01:) | |********************************************************************************************_| | DEPTH | 0.155 | 3.528 | 14.022 | 0.240 | 0.804 | 0.924 | 0.967 | | DEPTH_PP | 0.154 | 3.425 | 13.910 | 0.238 | 0.805 | 0.925 | 0.967 | | DEPTH_GT | 0.148 | 3.050 | 13.874 | 0.235 | 0.812 | 0.926 | 0.969 | | DEPTH_PP_GT | 0.147 | 2.972 | 13.770 | 0.233 | 0.814 | 0.927 | 0.969 | |*********************************************************************************************|
> 
> Can you provide some advice or help，thanks！

I got almost the same results. Have you fixed the problem?
